File: 1651-shuffle-string/shuffle-string.java
class Solution {
    public String restoreString(String s, int[] indices) {
        String st="";
        for(int i=0;i<indices.length;i++){
            for(int j=0;j<indices.length;j++){
                if(indices[j]==i){
                    st+=s.charAt(j);
                }
            }
        }
        return st;
    }
}

File: 1651-shuffle-string/shuffle-string.c
char* restoreString(char* s, int* indices, int indicesSize) {
     int len = strlen(s);
    char* result = (char*)malloc((len + 1) * sizeof(char)); 

    for (int i = 0; i < len; i++) {
        result[indices[i]] = s[i];  
    }

    result[len] = '\0'; 
    return result;
}